### DISCUSSION TITLE
[General] Examples of transactions with account creation

### DISCUSSION THREAD

**Discussion by akolotov** [zkSync-Community-Hub/zksync-developers#145] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

I has read about Account Abstraction in the docs and in order to understand how the deployment and further interaction with the custom accounts look like in the wild I would like to see examples of some accounts in zkSync Era explorer. 

### What is the specific issue or error you're encountering?

I went through thousands of transactions on https://explorer.zksync.io/address/0x0000000000000000000000000000000000008006 and https://goerli.explorer.zksync.io/address/0x0000000000000000000000000000000000008006 and cannot find any `createAccount`  or `create2Account` that I expect to see as [the documentation](https://era.zksync.io/docs/reference/concepts/account-abstraction.html#the-deployment-process).

### Can you share the error messages or logs you're receiving, if any?

NA

### Have you made any recent changes to the contract before encountering this issue?

NA

### Are there any external libraries or contracts that your contract interacts with?

NA

### Can you provide the relevant portions of your contract code where the issue is occurring?

NA

### Have you tried to isolate the problem, and if so, what were the results?

NA

### What steps have you already taken to try to resolve the issue?

Before I will implement a code to discover calls to `0x8006` which is definitely overhead, could you provide me few examples of custom accounts in the testnet or mainnet?

**Comment by caffeinum** [comment, marked as answer]
Here's my deployment:

https://goerli.explorer.zksync.io/address/0xEB64CA045DFd30A59BED7178Ca78995cA27AD677#transactions

This is AA Factory from tutorial https://era.zksync.io/docs/dev/tutorials/custom-aa-tutorial.html#the-factory

This is account creation tx: https://goerli.explorer.zksync.io/tx/0xafce2ebf4de275b417d21b52ebca893219b3c4e04aabf12a3c42e045e8cdeb61

This is the account itself:
https://goerli.explorer.zksync.io/address/0xf3Ef6f551329539aD075205A537441c92f778571

I wasn't able to verify the code on that, so there's a verified version of the same contract:
https://goerli.explorer.zksync.io/address/0xd56F72D87B4bF941bD9120db552C04901dBB3F45#contract

Edit:

I also recommend going through the tutorial and trying to execute the steps, even if you don't fully understand what's going on, it helps to start building an intuition

**Comment by dutterbutter** [comment]
Since all accounts on zkSync are smart accounts you can also review our [DefaultAccount.sol](https://github.com/matter-labs/era-system-contracts/blob/main/contracts/DefaultAccount.sol) contract which represents and behaves exactly like externally owned accounts.

**Comment by johnwesener** [comment]
Account Abstraction is a powerful feature of zkSync that allows for custom logic to be executed on the Ethereum blockchain. However, finding examples of custom accounts deployed on zkSync's testnet or mainnet can be challenging.

To locate such examples, you may need to perform targeted searches through zkSync's transaction explorer or examine relevant contracts deployed on Ethereum that interact with zkSync. Additionally, reaching out to the zkSync community or developers who have implemented custom accounts may provide valuable insights and examples.

Once you've identified examples of custom accounts, you can analyze their transaction history and interactions to gain a better understanding of how they are deployed and utilized in practice.

**Comment by natyfikapsha** [comment]
Seeking examples of custom accounts in zkSync Era explorer? Explore the transactions at address 0x0000000000000000000000000000000000008006 on both testnet and mainnet to observe createAccount and create2Account operations. Happy exploring!

**Comment by gauvinjohnny** [comment]
Unfortunately, as of now, there are no available examples of custom accounts in zkSync Era explorer. However, you can refer to the zkSync documentation and consider implementing code to discover calls to the specific account (0x0000000000000000000000000000000000008006) as a workaround to observe custom account interactions.

**Comment by moyemv** [comment]
The user is seeking examples of custom accounts in the zkSync Era explorer to better understand the deployment and interaction process of custom accounts in the zkSync protocol. The user has not reported encountering any specific issue or error and has not provided any relevant code or repository link. Further information may be needed to provide examples of custom accounts in the zkSync Era explorer.
